srcCpp03/perftest_publisher.cxx

```diff
@@ -966,6 +966,18 @@ void perftest_cpp::PrintConfiguration()
         stringStream << "Listeners\n";
     }
 
+    // Thread priority
```

Comment:
Unless you started implementing this for C++03, I would say that you added it to the wrong file.